docs/source/using/basic-usage.adoc

```diff
@@ -1,14 +1,14 @@
 include::variables.adoc[]
 
-= Managing {project}
+= Basic Usage
 :icons:
 :toc: macro
 :toc-title:
 :toclevels: 2
 
 toc::[]
 
-[[managing-minishift-overview]]
+[[basic-commands-overview]]
```

Comment:
Let's use `basic-usage-overview` to match with header name.

Reply:
true true

Reply:
fixed